<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AirportsLoadWindow</class>
 <widget class="QWidget" name="AirportsLoadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="3">
    <widget class="QTabWidget" name="tw_graps">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_year">
      <attribute name="title">
       <string>Год</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_month">
      <attribute name="title">
       <string>Месяц</string>
      </attribute>
      <widget class="QComboBox" name="cb_monthSelect">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="tl_AirportName">
     <property name="text">
      <string>Загруженность аэропорта</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>459</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>459</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pb_close">
     <property name="text">
      <string>Закрыть окно</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
